applications/nrf5340_audio/src/bluetooth/le_audio_cis_headset.c

```diff
@@ -301,6 +301,12 @@ static int lc3_config_cb(struct bt_conn *conn, const struct bt_audio_ep *ep, enu
 
 			*stream = audio_stream;
 			*pref = qos_pref;
+#if (CONFIG_BT_MCC)
+			ret = ble_mcs_discover(conn);
+			if (ret) {
+				LOG_ERR("Failed to start discovery of MCS: %d", ret);
+			}
+#endif /* CONFIG_BT_MCC */
```

Comment:
I find it a bit strange that this is in the `lc3_config_cb`. Sounds like this is only done for lc3, but it should be done for any codec.

Reply:
As discussed, this will be handled in a separate task